File: sound-2024/konsolowa/Album.java
public class Album {
    public String artist;
    public String album;
    public int songsNumber;
    public int year;
    public int downloadNumber;

    public Album(String artist, String album, int songsNumber, int year, int downloadNumber) {
        this.artist = artist;
        this.album = album;
        this.songsNumber = songsNumber;
        this.year = year;
        this.downloadNumber = downloadNumber;
    }

    @Override
    public String toString() {
        return artist + "\n" +
                album + "\n" +
                songsNumber + "\n" +
                year + "\n" +
                downloadNumber + "\n";
    }
}

File: sound-2024/desktopowa/Album.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DesktopowaApk.Models
{
    public class Album
    {
        public string artist { get; set; }
        public string album { get; set; }
        public int songsNumber { get; set; }
        public int year { get; set; }
        public int downloadNumber { get; set;  }

        public Album(string artist, string album, int songsNumber, int year, int downloadNumber)
        {
            this.artist = artist;
            this.album = album;
            this.songsNumber = songsNumber;
            this.year = year;
            this.downloadNumber = downloadNumber;
        }
    }
}
